### PR TITLE
Rename "Crate" to "CrateDB", and adjust a few more texts

### PR DESCRIPTION
**Pull request by amotl**
## About

A few more text edits suggested at GH-819, and more.

### CODE REVIEW

**Review by matriv — APPROVED** [review]
LGTM, but why did you remove those entries from the `LICENCE`?
Maybe wait for another approval for this ^^ ?

**Comment by amotl** [comment]
> Why did you remove those entries from the LICENSE file?

I've trimmed a few of those in other repositories already. In this case, I just moved it to GH-824, in order to discuss later.

**Pull request merged**
